runtime/parachains/src/disputes/tests.rs

```diff
@@ -788,16 +840,13 @@ fn test_freeze_on_note_included() {
 					),
 				),
 				(
-					DisputeStatement::Valid(ValidDisputeStatementKind::Explicit),
+					DisputeStatement::Valid(ValidDisputeStatementKind::BackingValid(
+						inclusion_parent,
+					)),
 					ValidatorIndex(1),
-					v1.sign(
-						&ExplicitDisputeStatement {
-							valid: true,
-							candidate_hash: candidate_hash.clone(),
-							session: 3,
-						}
-						.signing_payload(),
-					),
+					v0.sign(&CompactStatement::Valid(candidate_hash).signing_payload(
```

Comment:
These changes were required in the previous version of the PR, when backing votes must be present always.
Now they are not strictly required, but I've kept these changes anyway as they don't hurt.